src/server/.tests/routes/controllers/logInUser.spec.js

```diff
@@ -0,0 +1,38 @@
+const logInUser = require('../../../routes/controllers/logInUser')
+const sinon = require('sinon')
+const assert = require('chai').assert
+
+let suite = {}
+describe('logInUser', () => {
+	beforeEach(() => {
+		suite = {}
+		suite.passportMock = createPassportMock()
+		suite.loginUserMock = logInUser(suite.passportMock)
+	})
+
+	it('should call passport authentication', () => {
```

Comment:
What's sense of testing if the function has been called? It could be called and did nothing to it.
Please rewrite it into a test that actually checks if functionality hasn't been broken.

Reply:
ok